api/src/wfl/api/routes.clj

```diff
@@ -157,15 +165,15 @@
    (merge
     exception/default-handlers
     {;; ex-data with :type :wfl/exception
-     ::workloads/invalid-pipeline          (partial exception-handler 400 "")
-     ::workloads/workload-not-found        (partial exception-handler 404 "")
-     UserException                         (partial exception-handler 400 "")
+     ::workloads/invalid-pipeline          (partial logging-exception-handler 400 "Invalid Pipeline." {:exception-type "InvalidPipeline"} :warning true)
+     ::workloads/workload-not-found        (partial logging-exception-handler 404 "Workload Not Found." {:exception-type "WorkloadNotFound"} :warning true)
+     UserException                         (partial logging-exception-handler 400 "Request Invalid." {:exception-type "UserException"} :warning true)
      ;; SQLException and all its child classes
-     SQLException                          (partial logging-exception-handler 500 "SQL Exception")
+     SQLException                          (partial logging-exception-handler 500 "An internal error has occurred during this request. The development team has been notified of this error." {} :error false)
      ;; handle clj-http Slingshot stone exceptions
-     :clj-http.client/unexceptional-status (partial exception-handler 400 "HTTP Error on request")
+     :clj-http.client/unexceptional-status (partial logging-exception-handler 400 "HTTP Error on request" {:exception-type "HTTPError"} :warning true)
      ;; override the default handler
-     ::exception/default                   (partial logging-exception-handler 500 "Internal Server Error")})))
+     ::exception/default                   (partial logging-exception-handler 500 "An internal error has occurred during this request. The development team has been notified of this error." {} :error false)})))
```

Comment:
Not sure we should commit to `The development team has been notified of this error.` ;-)